__tests__/app.test.js

```diff
@@ -13,29 +13,73 @@ afterAll(() => {
 });
 
 describe("GET /api/categories", () => {
-    test("GET - status:200, responds with an array of categories objects", () => {
-      return request(app)
-        .get("/api/categories")
-        .expect(200)
-        .then(({ body }) => {
-          const { categories } = body;
-          expect(categories).toHaveLength(4);
-          categories.forEach((category) => {
-            expect(category).toEqual(
-              expect.objectContaining({
-                slug: expect.any(String),
-                description: expect.any(String),
-              })
-            );
-          });
+  test("GET - status:200, responds with an array of categories objects", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        const { categories } = body;
+        expect(categories).toHaveLength(4);
+        categories.forEach((category) => {
+          expect(category).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
         });
-    });
-    test("GET - status:404, not found", () => {
-        return request(app)
-          .get("/api/badurl")
-          .expect(404)
-          .then((res) => {
-            expect(res.body.msg).toBe("Invalid URL!");
-          });
       });
-});
\ No newline at end of file
+  });
+  test("GET - status:404, not found", () => {

```

Comment:
Nothing wrong with this test, but as the 404 bad endpoint error isn't tied to the /api/categories endpoint, this should be moved to its own describe block.